### PR TITLE
Chore/update assets for docs

### PR DESCRIPTION
**Pull request by Huongg**
> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to a separate `kedro-datasets` package in
> [`kedro-plugins` repository](https://github.com/kedro-org/kedro-plugins). Any changes to the dataset implementations
> should be done by opening a pull request in that repository.
## Description
To upload the new UI branding screenshots for the Kedro docs. [Kedro-viz #1365](https://github.com/kedro-org/kedro-viz/issues/1365)

## Development notes
Updated all the screenshots + gifs with the new UI

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Docs failing:

```
sphinx.errors.SphinxWarning: /home/docs/checkouts/readthedocs.org/user_builds/kedro/checkouts/2625/docs/source/index.rst:broken link: https://raw.githubusercontent.com/kedro-org/kedro/chore/update-docs-font-family/static/img/kedro_banner.png (404 Client Error: Not Found for url: https://raw.githubusercontent.com/kedro-org/kedro/chore/update-docs-font-family/static/img/kedro_banner.png)
```

File: docs/source/index.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
    contain the root `toctree` directive.
 
 
-.. image:: https://raw.githubusercontent.com/kedro-org/kedro/develop/static/img/kedro_banner.png
+.. image:: https://raw.githubusercontent.com/kedro-org/kedro/chore/update-docs-font-family/static/img/kedro_banner.png
```
Comment (stichbury) [inline docs/source/index.rst:7]: Is this the permanent home? It's probably really picky but it would be nice if the URL didn't include "chore" 😆

Comment (stichbury) [reply docs/source/index.rst:7]: Awesome, sounds like a plan!

Comment (Huongg) [reply docs/source/index.rst:7]: oh actually this is from the other branch, I'll remove it

**Comment by astrojuanlu** [comment]
Spotted an `autoreload.gif` that still uses the old theme: https://kedro--2625.org.readthedocs.build/en/2625/visualisation/kedro-viz_visualisation.html#automatic-visualisation-updates not sure if there are more

**Comment by stichbury** [comment]
In the section that @astrojuanlu flags (https://kedro--2625.org.readthedocs.build/en/2625/visualisation/kedro-viz_visualisation.html#automatic-visualisation-updates) there is a pretty large graphic (long vertical drop) and I'm wondering if it would be sensible for me to go through and resize all the images to either a fixed height or width. I'll ask @Mackay031 what he recommends.

Also, am I right in thinking there is a plugin for Sphinx where if you have an image relatively small, you can click on it and it opens it larger so you can review detail? That may be a good option to add so we can reduce sizes to a common maximum for easy scrolling and reading, whilst retaining details.

HOWEVER, neither of these points affect the PR, and I'll make a separate issue to consider whether to revise diagram sizes.

I'm happy to pass the autoreload gif too -- it's just the font and I don't think it's significant.

**Review by tynandebold — APPROVED** [review]

**Comment by Mackay031** [comment]
I think we should consider a lightbox/modal to show a larger image.

**Pull request merged**